util/config/fixpath.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import os
+from sys import argv
+
+
+def splitescaped(s, delim=":", escape='\\'):
```

Comment:
I probably would do with with something like:

```python
pattern = r'(?<!{0}){1}'.format(escape, delim)
re.split(pattern, s)
```
this wont support escaping your escape character (same as your way, which is probably fine), but I find it much easier to read.